kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/base/BaseTest.java

```diff
@@ -1,118 +1,118 @@
+        screenshooter.compareScreenshots(name, shouldUpdateScreenshots);
+    }
 
-  static {
-    if (!new File("./.env").exists()) {
-      try {
-        FileUtils.copyFile(new File(".env.example"), new File(".env"));
-      } catch (IOException e) {
-        log.error("couldn't copy .env.example to .env. Please add .env");
-        e.printStackTrace();
-      }
+    @BeforeAll
+    public static void start() {
+        Testcontainers.exposeHostPorts(8080);
+        webDriverContainer.start();
+        webDriverContainer.isRunning();
+        RemoteWebDriver remoteWebDriver = webDriverContainer.getWebDriver();
+        WebDriverRunner.setWebDriver(remoteWebDriver);
     }
-    Dotenv.load().entries().forEach(env -> System.setProperty(env.getKey(), env.getValue()));
-    if (TestConfiguration.CLEAR_REPORTS_DIR) {
-      clearReports();
+
+    static {
+        if (!new File("./.env").exists()) {
+            try {
+                FileUtils.copyFile(new File(".env.example"), new File(".env"));
+            } catch (IOException e) {
+                log.error("couldn't copy .env.example to .env. Please add .env");
+                e.printStackTrace();
+            }
+        }
+        Dotenv.load().entries().forEach(env -> System.setProperty(env.getKey(), env.getValue()));
+        if (TestConfiguration.CLEAR_REPORTS_DIR) {
+            clearReports();
+        }
+        setup();
     }
-    setupSelenoid();
-  }
 
-  @AfterAll
-  public static void afterAll() {
-//    closeWebDriver();
-//    selenoid.close();
-  }
 
-  @SneakyThrows
-  private static void setupSelenoid() {
-    String remote = TestConfiguration.SELENOID_URL;
-    if (TestConfiguration.SHOULD_START_SELENOID) {
-      selenoid.start();
-      remote =
-          "http://%s:%s/wd/hub"
-              .formatted(selenoid.getContainerIpAddress(), selenoid.getMappedPort(4444));
+    @AfterEach
+    public void afterMethod() {
+        webDriverContainer.getWebDriver().manage().deleteAllCookies();
+        Allure.addAttachment("Screenshot",
+                new ByteArrayInputStream(((TakesScreenshot) webDriverContainer.getWebDriver()).getScreenshotAs(OutputType.BYTES)));
     }
 
-    Configuration.reportsFolder = TestConfiguration.REPORTS_FOLDER;
-    if (!TestConfiguration.USE_LOCAL_BROWSER) {
-      Configuration.remote = remote;
-      TestConfiguration.BASE_URL =
-          TestConfiguration.BASE_URL.replace("localhost", "host.docker.internal");
-    }
-    Configuration.screenshots = TestConfiguration.SCREENSHOTS;
-    Configuration.savePageSource = TestConfiguration.SAVE_PAGE_SOURCE;
-    Configuration.reopenBrowserOnFail = TestConfiguration.REOPEN_BROWSER_ON_FAIL;
-    Configuration.browser = TestConfiguration.BROWSER;
-    Configuration.baseUrl = TestConfiguration.BASE_URL;
-    Configuration.browserSize = TestConfiguration.BROWSER_SIZE;
-    var capabilities = new DesiredCapabilities();
-//    DesiredCapabilities capabilities = DesiredCapabilities.chrome();
```

Comment:
?

Reply:
baseTest class wrote with WebBrowserContainer and removed all the rest